src/scrabble.js

```diff
@@ -1,5 +1,109 @@
-function scrabble() {
+const lettersToPoints = { // Object containing each letter in uppercase as the key, and their respective values as the value
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10
+}
+// Object containing each type of bracket as the key, whose value is another object containing information about said bracket 
+// (multiplier value, corresponding open/closing bracket, and what type of bracket it is (open or closing))
+const brackets = { 
+  '{': {multi: 2, corresponding: '}', type: 'open'},
+  '[': {multi: 3, corresponding: ']', type: 'open'},
+  '}': {multi: 2, corresponding: '{', type: 'close'},
+  ']': {multi: 3, corresponding: '[', type: 'close'}
+}
```

Comment:
This is new to me! Will definately have to look into this. Very clean

Reply:
After reading this , I think i will start using more objects to code!